discovery-provider/src/tasks/entity_manager/user.py

```diff
@@ -133,3 +149,31 @@ def update_user(params: ManageEntityParameters):
     params.add_user_record(user_id, user_record)
 
     return user_record
+
+
+def verify_user(params: ManageEntityParameters):
+    validate_user_tx(params)
+
+    user_id = params.entity_id
+    existing_user = params.existing_records[EntityType.USER][user_id]
+    existing_user.is_current = False  # invalidate
```

Comment:
nit i don't think this is necessary since invalidation happens before saving
https://github.com/AudiusProject/audius-protocol/blob/master/discovery-provider/src/tasks/entity_manager/entity_manager.py#L192

Reply:
Copied this part from the update flow